.github/ISSUE_TEMPLATE/package.yml

```diff
@@ -0,0 +1,60 @@
+name: Packages
+description: Report problems and suggest ideas for the packages
+labels: [package]
+
+body:
+  - type: markdown
+    attributes:
+      value: >-
+        Before opening a new issue, make sure it isn't covered by an existing issue.
+        Please search for [issues with the `package` label][package-issues] (including
+        closed issues) before you continue.
+
+
+        [package-issues]:
+        https://github.com/simple-icons/simple-icons/issues?q=is%3Aissue+label%3Apackage+is%3Aopen
+
+  - type: dropdown
+    attributes:
+      label: Kind of issue
+      description: >-
+        If your issue type is not here, select "other" and explain in the
+        "Description" field below.
+      options: [improvement, bug, other]
+    validations:
+      required: true
+
+  - type: dropdown
+    attributes:
+      label: Which package does this issue concern?
+      options: [NPM, Packagist]
+      multiple: true
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Description
+      description: |
+        Anything relevant, for example:
+        - For bugs: "Steps to reproduce" and "Expected behavior"
+        - For improvements: An example of a use case
+        - Package issues: Version information
+          - For JavaScript/NodeJS: `node --version`
+          - For PHP/Packagist: `composer --version`
+        - etc.
+      placeholder: "Example: The NPM package does not work for my use case..."
+    validations:
+      required: true
+
+  - type: dropdown
+    attributes:
+      label: Contributing
+      description: >-
+        This is an open source project and we welcome contributions. Do you want to
+        work on this issue?
+      options:
+        - "Yes"
+        - "No"
+    validations:
+      required: true
```

Comment:
Not sure if this makes sense here, I think bug reports will require some back and forth before you start working on them. This is opposed to, in particular, icon requests and icon updates where we often ask "Do you want to work on this?" in the first comment.

(arguably this also goes for documentation, but at least to me it does seem sensible to have this field there)

Reply:
That makes sense, do you think we should just remove this field?

Reply:
I would say so yes. If we find it's missing we can always add it later.